api/migrations/0001_initial.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-10-03 03:16
+from __future__ import unicode_literals
```

Comment:
This is an auto generated file. I think I removed it from the other ones, but this line will be added to every migration file generated, not sure if it's worth removing everytime